filament/src/materials/colorGrading/colorGradingAsSubpass.mat

```diff
@@ -36,7 +36,7 @@ material {
     ],
     outputs : [
         {
-            name : color,
+            name : dummy,
```

Comment:
Why do we still need this output?

Reply:
Ah yes... we do. Because matc crashes without it. This is something I wanted to investigate next.

Reply:
Hm, yeah that sounds like a bug.